### PR TITLE
chore: adapting ndbx styles

### PR DESCRIPTION
**Pull request by ultimatekrishna**
**What I did**
Adapted css as close as possible to ndbx
**Related issue**

### CODE REVIEW

File: _layouts/assets/main.css [hunk 1]
```diff
@@ -40,14 +40,19 @@ main.with-side-nav { left: 240px; }
 main .page-content { padding: 0px 0px 0px 0px; }
 
 /** Content elements */
-.page-content {
-  color: rgb(65,65,65);
+.page-content p{
```
Comment (cheld) [inline _layouts/assets/main.css:43]: maybe we should have a different top level div for index and documentation?  or an additional?
```
<div class="page-content">
   <div class="welcome-page"> or <div class="documentation-page"> 
      xxxxx
  </div>
<div>
```
So, we are more flexible with css selections. Or do we want to be consistent anyway?

File: _layouts/default.html [hunk 2]
```diff
@@ -14,7 +14,10 @@
   <!-- Set flags used to render top menu and side navidation -->
   {% if page.url == '/' %} 
     {% assign show-sidenav = false %} 
-    {% assign welcome-menu = 'active' %} 
+    {% assign welcome-menu = 'inactive' %}
```
Comment (cheld) [inline _layouts/default.html:17]: I did not understand this code. From reading code I would assume that when page is loaded no menu is selected. 

However, I could not spot a difference to previous behavior when running the website. I could also not spot a different behavior to ndbx

File: _pages/index/index.html [hunk 3]
```diff
@@ -109,39 +57,89 @@ <h1 class="purpose-headline">
                         type="image/svg+xml" style="height:50px"></object>
                 </div>
                 <h3 class="">Engage</h3>
-                <span class="">
+                <p class="">
```
Comment (cheld) [inline _pages/index/index.html:60]: Empty class?

File: _pages/index/index.html [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ---
-permalink: /
+permalink: /index
```
Comment (cheld) [inline _pages/index/index.html:2]: The permalink and the file location in git are totally independant. In other words you can change permalink without moving the file or you can move the file without changing the permalink.

What are you trying to achieve?

File: _layouts/default.html (hunk 2 above)
Comment (cheld) [inline _layouts/default.html:17]: Change should not be needed when not changing permalink, I think

File: _pages/index/index.html
Comment (cheld) [inline _pages/index/index.html:1]: Please rename index folder to 'welcome'. (No code changes required as it no effect on running application)

File: _layouts/assets/main.css (hunk 1 above)
Comment (ultimatekrishna) [reply _layouts/assets/main.css:43]: I would go for consistancy for now and later see if there are any special cases. but feel free to adapt if you see something special.

**Pull request merged**